pyproject.toml

```diff
@@ -41,3 +41,10 @@ select = [
 
 [tool.ruff.lint.per-file-ignores]
 "*.pyi" = ["F403", "F405", "PYI001", "PYI002"]
+
+[tool.cibuildwheel]
+skip = ["*-musllinux_i686"]
+
+[tool.cibuildwheel.linux]
```

Comment:
Qiskit Core relies on [dtolnay/rust-toolchain](https://github.com/dtolnay/rust-toolchain) to setup the rust toolchain. Do you think it makes the workflow simpler/reliable?

https://github.com/Qiskit/qiskit/blob/dd145b54be9d583d3857edfbd81e910600249adf/.github/workflows/wheels.yml#L29-L31

Reply:
I don't see the need to do that. I got this from https://cibuildwheel.pypa.io/en/stable/faq/#building-rust-wheels.